frontend/chat-plugin/src/components/websocket/index.ts

```diff
@@ -1,7 +1,7 @@
 import {Client, messageCallbackType, IFrame} from '@stomp/stompjs';
 import 'regenerator-runtime/runtime';
 import {start, getResumeToken, sendMessage} from '../api';
-import {Text} from 'types';
+import {SuggestionResponse, Text} from 'types';
```

Comment:
See my comment on the `content.ts` file: We should move any types to the chatplugin itself